controllers/storagecluster/cephblockpools.go

```diff
@@ -18,29 +18,6 @@ import (
 
 type ocsCephBlockPools struct{}
 
-// ensures that peer cluster secret exists and adds it to CephBlockPool
-func (o *ocsCephBlockPools) addPeerSecretsToCephBlockPool(r *StorageClusterReconciler, storageCluster *ocsv1.StorageCluster, poolName, poolNamespace string) *cephv1.MirroringPeerSpec {
```

Comment:
This commit is having 3 different changes? can we move specific changes to a commit (with details) so that it helps in reviewing and captures the history on why its changed